tools/wptrunner/wptrunner/formatters.py

```diff
@@ -50,6 +53,37 @@ def replace_lone_surrogate(data):
     return LONE_SURROGATE_RE.subn(surrogate_replacement, data)[0]
 
 
+def convert_reftest_screenshots(extra):
```

Comment:
Can this go in `executors/base.py` as part of `convert_reftest_result`? That way we only do it once per result, not once per formatter.

Reply:
Yeah, I've moved it into `executors/base.py` as part of `reftest_result_converter` which will be called regardless of whether Marionette internal reftest runner is used (originally I didn't think it was possible because I was looking at `RefTestImplementation`, which isn't used by the internal reftest runner).